components/eventing-controller/README.md

```diff
@@ -33,6 +45,16 @@ This section explains how to use the Eventing Controller.
     | reconcile-period        | The period between triggering of reconciling calls. | 10 minutes    |
     | enable-debug-logs       | Enable debug logs.                                  | false         |
 
+- The Nats Controller comes with the following command line argument flags:
+
+    | Flag                    | Description                                         | Default Value |
+    | ----------------------- | --------------------------------------------------- | ------------- |
+    | metrics-addr            | The address the metric endpoint binds to.           | :8080         |
```

Comment:
```suggestion
    | `metrics-addr`            | The address the metric endpoint binds to.           | `:8080`         |
```

Reply:
As per guidelines, we use `code font` for flags (values in tables as well) and **bold** envs.  Could you pls adjust all of them in tables?